Add Refund files

julionc opened the pull request:
/cc @andrewpaliga @ch33sybr3ad

andrewpaliga commented:
I actually don't know enough about this to review. @dominiquesr?

julionc commented:
/cc @kevinhughes27

dominiquesr commented on test/refund_test.rb:
```diff
@@ -0,0 +1,22 @@
+    fake "orders/450789469/refunds/509562969", :method => :get, :body => load_fixture('refund')
+    refund = ShopifyAPI::Refund.find(509562969, :params => {:order_id => 450789469})
+    assert_equal 509562969, refund.id
+  end
```
Should there be a test for calculate as well?

dominiquesr commented on test/fixtures/refund.json:
```diff
@@ -0,0 +1,116 @@
+              "value": "Happy Birthday"
+            },
+            {
+              "name": "Custom Engraving Back",
```
I don't think you'd ever put Merry Christmas on the same item as Happy Birthday, but maybe I'm the only one that cares that this example doesn't make sense. ;)

julionc commented:
You're completely right, @dominiquesr :sweat_smile:
Maybe, someone should update the json response from documentation. (See [refund#show](https://docs.shopify.com/api/reference/refund#show) & [order#show](https://docs.shopify.com/api/reference/order#show)).
BTW, i just removed Merry Christmas property, and added calculate method.

dominiquesr commented:
LGTM but @kevinhughes27 has more experience in this repo.

kevinhughes27 commented:
I'm fine with this.

julionc commented:
Bump
